governance-proposal/TSC-Charter-Draft.md

```diff
@@ -0,0 +1,103 @@
+
+## Section 3. Board’s Role in Setting Node.js Foundation’s Strategic Direction.  
+
+The Board will set the overall TSC Policy. The policy will describe the overarching scope of the Node.js Foundation initiative, Node.js Foundation’s technical vision and direction and project release expectations in the form of expected cadence and intent. The Board will use the TSC as a delegate body for governing technical implementation, individual project scope and direction while they remain within the scope and direction of the policies as described in the TSC Policy document and approved by the Board.
+
+## Section 4. Establishment of the TSC.  
+
+TSC memberships are not time-limited. There is no fixed size of the TSC. However, the expected target is between 6 and 12, to ensure adequate coverage of important areas of expertise, balanced with the ability to make decisions efficiently.
+
+There is no specific set of requirements or qualifications for TSC membership beyond these rules. The TSC may add additional members to the TSC by a standard TSC motion and vote.  A TSC member may be removed from the TSC by voluntary resignation, or by a standard TSC motion.
+
+Changes to TSC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+No more than one-fourth of the TSC members may be affiliated with the same employer. If removal or resignation of a TSC member, or a change of employment by a TSC member, creates a situation where more than one-fourth of the TSC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more TSC members affiliated with the over-represented employer(s).
+
+The TSC members shall consist of Maintainers from Core Projects as defined in the project lifecycle document and Section 7.
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+## Section 5. Responsibilities of the TSC.  
+
+Subject to such policies as may be set by the Board, the TSC is responsible for all technical development within the Node.js Foundation, including:
+
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Project governance and process (including this policy)
+* GitHub repository hosting
```

Comment:
Is it appropriate to specify 'GitHub' here? Should this just be 'Repository hosting'?